tox.ini

```diff
@@ -129,7 +129,8 @@ exclude =
     __init__.py
 max-line-length = 88
 # The following conflict with `black` which is the more pedantic.
-ignore =
+extend-ignore =
+    B023
```

Comment:
B023 has only one instance and it's from https://github.com/opencobra/cobrapy/blob/7b1a4ecaa5df90546f760bd100581f7cdf174d89/src/cobra/io/mat.py#L915 due to variable binding outside of scope in the lambda. I couldn't think of a better way to fix it while keeping the functionality. Maybe we'll find a better way soon but since it's not exactly hurting either way, I chose to exclude the check.

Reply:
I think you can add a comment to the line:

```py
group_members = model.reactions.query(lambda x: x.subsystem == g_name)  # noqa: B023
```

rather than ignore it for the whole project.

Reply:
Okay yes makes sense, will make the change.

Reply:
@Midnighter The inline comment for ignoring doesn't work for B023, what do you suggest?

Reply:
I pushed a fix (I think).